web/app/integration/specs/namespace-select.js

```diff
@@ -0,0 +1,114 @@
+const assert = require('assert');
```

Comment:
Hmm, I am not quite sure how this ends up using the chai assertion library we imported in `package.json`. Is there some underlying code that does some magic for these tests to use chai under the hood?

Reply:
Thanks for the review! Chai is the assertion library that is used in conjunction with Mocha. We initialize it in the [before hook of wdio.conf.js](https://webdriver.io/docs/frameworks.html). ⭐️